packages/helm-charts/blockscout/templates/blockscout-api.ingress.yaml

```diff
@@ -19,13 +19,16 @@ metadata:
 spec:
   tls:
   - hosts:
-    - {{ .Release.Name }}-api.{{ .Values.domain.name }}.org
+    - {{ .Release.Name }}-api.{{ .Values.blockscout.domainName }}
     secretName: {{ .Release.Name }}-api-tls
   rules:
-  - host: {{ .Release.Name }}-api.{{ .Values.domain.name }}.org
+  - host: {{ .Release.Name }}-api.{{ .Values.blockscout.domainName }}
```

Comment:
`domainName` now includes the tld whilst previously it was just the host?

Reply:
Yeah, I don't have a good explanation why we need to hardcode the `.org` part.

Reply:
My impressing is that originally the domain could be the same as the like `project_name` but eventually this was broke.